codersdk/deployment.go

```diff
@@ -804,8 +804,12 @@ func DefaultSupportLinks(docsURL string) []LinkConfig {
 	}
 }
 
+func RemoveTrailingVersionInfo(v string) string {
```

Comment:
I would suggest...

- making this function private (lowercase)
- making a new deployment_internal_test.go file that declares `package codersdk` (and not `codersdk_test`)
- moving the tests you added in there